src/main.js

```diff
@@ -118,6 +122,7 @@ makePosterButton.addEventListener('click', inputFormView);
 randomPosterButton.addEventListener('click', makeNewRandomPoster);
 returnToMainPageButton.addEventListener('click', returnToMainPage);
 backToMainButton.addEventListener('click', returnToMainPage);
+showPosterButton.addEventListener('click', createCustomPoster);
```

Comment:
What would happen if we prevented the automatic reloading of the main page in the createCustomPoster function?